lib/libc/Kconfig

```diff
@@ -678,3 +678,10 @@ config LIB_USRWORKSTACKSIZE
 
 endif # LIB_USRWORK
 endif # BUILD_PROTECTED || BUILD_KERNEL
+
+
+config AUDIO
```

Comment:
No, you can not move file (lib_buffer.c) from libc to framework. Because lib_buffer.c is calling from device driver (audio_ioctl).
In other words, if you move file to framework, it will be a user context. But device driver is a kernel context.
So, this folder or file can not be a top layer for audio framework, and it does not have controlling device driver. Only for this functionality, audio is too big title.